.ci_support/build_all.py

```diff
@@ -0,0 +1,48 @@
+import conda_build.conda_interface
+import networkx as nx
+import conda_build.api
+from compute_build_graph import construct_graph
+import argparse
+import os
+
+def build_all(recipes_dir, arch, variant_config_files):
+    channel_urls=['local', 'conda-forge', 'defaults']
+    index = conda_build.conda_interface.get_index(channel_urls=channel_urls)
+    conda_resolve = conda_build.conda_interface.Resolve(index)
+
+    config = conda_build.api.Config(variant_config_files=variant_config_files, arch=arch,
+                                    channel_urls=channel_urls)
+
+    worker={'platform': get_host_platform(), 'arch': arch, 'label': get_host_platform()}
+
+    G = construct_graph(recipes_dir, worker=worker, run='build', conda_resolve=conda_resolve,
+                        folders=os.listdir(recipes_dir), config=config)
+
+    order = list(nx.topological_sort(G))
+    order.reverse()
```

Comment:
Why is the build order reversed? (wouldn't this build the bottom of the graph first?)

Reply:
Could be that I build the graph backward when adding the nodes.  I have considered going back and reversing it, but haven't found time.  When I have tried, it ended up exploding in my face.

Reply:
It depends on how the edge direction is defined. reverse here gives the order that I need